video-player-in-flutter-using-chewie-and-getx/index.md

```diff
@@ -0,0 +1,287 @@
+
+We'll take advantage of the first library, `Chewie` to play videos and audio files, and the second one, `GetX`, for downloading them from Youtube's  API.
+
+### Prerequisites
+
+- Visual Studio or any code editor installed.
+- Have an Understanding of Flutter Widgets
+- Familiar with dart programming languages.
+- Knowledge in Flutter.
+
+We're going to create a screen to show our video player. This screen will have buttons for controlling the playback and two text fields to write the URL of the resource we want to play and its title.Our videos will be fetched from the internet rather than playing the local videos in our phones.
+
+#### VideoScreen
+This is the homepage of our Video player application.This is the page that we will use to interact with the our videos.Let's start with the a basic page with a title.Our Title will be positioned at the top of our page and will be contained in the Appbar Widget.
+
+```dart 
+
+class VideoScreen extends StatefulWidget { 
+
+@override
+
+VideoScreenState createState() => new VideoScreenState();
+
+} 
+
+class VideoScreenState extends State { 
+
+@override 
+
+Widget build(BuildContext context) { 
+
+ return new Scaffold(//contains our components of the application.
+
+     appBar: new AppBar( 
+
+       title: new Text("Video Player"), 
+
+     ),
+
+   );
+
+ } 
+
+}
+
+```
+
+As you see, we have a simple screen with an `appbar `, which will have the title of our player, and a `Scaffold`. The `Scaffold` will contain the background image of our video player and two rows: one with the "Play" button on top and another showing the title and URL of the selected resource.Play button will be used to execute the streaming of our video.The Title and URL are just simply for identification purposes of our video just to be sure which video is playing at that time.
+
+#### Align Widget
+This widget contains a container where we will add our components. The main purpose of the align widget is to make sure that the compononets will be placed correctly and in a more appealing location on the screen.
+
+```dart
+
+Widget build(BuildContext context) { 
+
+ return new Align( alignment: Alignment.center, //align the child at the center of the screen.
+
+ child: new Padding( 
+
+   padding: const EdgeInsets.all(16.0),//all edges will be 16 pixels from the walls.
+
+   child: new Container( 
+
+     height: 300.0, width: 300.0, 
+
+     decoration: new BoxDecoration( color: Colors.black45),
+
+   ), 
+
+ ), 
+
+ ); 
+
+}
+
+```
+
+We use the `Align` widget to align the element in the `Center` of the screen. In this case, it will be a `Container`, so we'll add a white background and a black border with inner padding of 16dp.The choice of the colour is so that the video can be seen more clearly regardless of the video quality.
+
+#### Row widget
+The Row widget will contains buttons of applications. This widget holds components from the top to bottom or vice versa.
```

Comment:
Change to " The Row widget will contain buttons of applications. This widget holds components from top to bottom or vice versa."